biometricks/src/main/java/com/handstandsam/biometricks/Biometricks.kt

```diff
@@ -168,129 +162,9 @@ sealed class Biometricks {
             showLoading: (Boolean) -> Unit
         ): BiometricPrompt.CryptoObject {
             val activity = fragment.requireActivity()
-
-            ensureFocus(activity)
-
-            return handleApi28Loading(activity, showLoading) {
-                suspendCoroutine<BiometricPrompt.CryptoObject> { continuation ->
-                    BiometricPrompt(
-                        fragment,
-                        ContextCompat.getMainExecutor(fragment.requireContext()),
-                        Callback(continuation)
-                    ).authenticate(promptInfo, cryptoObject)
-                }
-            }
+            return showPrompt(activity, cryptoObject, promptInfo, showLoading)
```

Comment:
Unfortunately you can't quite do this because the `BiometricPrompt` call needs a fragment in this case so it's scoped correctly.

Reply:
Ahh crap, okay.  Should have let you review first. You know a lot of these really 💩 issues.